File: server/src/main/java/com/fours/humorme/HumorMeApplication.java
package com.fours.humorme;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import springfox.documentation.swagger2.annotations.EnableSwagger2;

@SpringBootApplication
@EnableSwagger2
public class HumorMeApplication {

	public static void main(String[] args) {
		SpringApplication.run(HumorMeApplication.class, args);
	}

}

File: final project/humorme/src/main/kotlin/com/example/humorme/HumormeApplication.kt
package com.example.humorme

import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.autoconfigure.domain.EntityScan
import org.springframework.boot.runApplication

@SpringBootApplication
@EntityScan(basePackages = arrayOf("com.example.humorme.model"))
class HumormeApplication

fun main(args: Array<String>) {
	runApplication<HumormeApplication>(*args)
}


